MSOLSpray.py

```diff
@@ -25,6 +25,7 @@
 parser.add_argument("-f", "--force", action='store_true', help="Forces the spray to continue and not stop when multiple account lockouts are detected.")
 parser.add_argument("--url", default="https://login.microsoft.com", help="The URL to spray against (default is https://login.microsoft.com). Potentially useful if pointing at an API Gateway URL generated with something like FireProx to randomize the IP address you are authenticating from.")
 parser.add_argument("-v", "--verbose", action="store_true", help="Prints usernames that could exist in case of invalid password")
+parser.add_argument("-s", "--sleep", help="Sleep this many seconds between tries")
```

Comment:
Would be nice if you could set default to 0 here and cast to int to avoid doing a `int(sleep)` later,
```suggestion
parser.add_argument("-s", "--sleep", default=0, type=int, help="Sleep this many seconds between tries")
```